### ISSUE TITLE
Handle connection review in a single interaction in wallet API v2

### ISSUE THREAD

**Issue by ValentinTrinque** [vegaprotocol/vega#6376]
Today, during the wallet connection review, we send two events:
- ConnectionReview
- WalletSelection

This creates a spaghetti workflow in parallel requests management, so we have to make it just one.

**Comment by ValentinTrinque** [comment]
After discussing about it with @notbot00 , it reminded me the reason of the separation. 

Internally, we talked about supporting permanent trust for some hostname connection. It's not implemented, but having two events allow a clean workflow, whereas, with a single one we can't know if the connection is automatically approved or not.


On hold in the meantime.

**Comment by ValentinTrinque** [comment]
Blocked until we have more insights on:
- https://github.com/vegaprotocol/vega/issues/6380
